internal/services/recoveryservices/backup_policy_vm_resource.go

```diff
@@ -517,6 +538,24 @@ func expandBackupProtectionPolicyVMRetentionWeeklyFormat(block map[string]interf
 	return &weekly
 }
 
+func flattenBackupProtectionPolicyVMResourceGroup(rpDetail *backup.InstantRPAdditionalDetails) []interface{} {
+	if rpDetail == nil || rpDetail.AzureBackupRGNamePrefix == nil {
+		return nil
+	}
+
+	block := map[string]interface{}{}
+
+	if rpDetail.AzureBackupRGNamePrefix != nil {
+		block["prefix"] = *rpDetail.AzureBackupRGNamePrefix
+	}
+
+	if rpDetail.AzureBackupRGNameSuffix != nil {
+		block["suffix"] = *rpDetail.AzureBackupRGNameSuffix
+	}
```

Comment:
These need to have a value set into state
```suggestion
	prefix := ""
	if rpDetail.AzureBackupRGNamePrefix != nil {
		prefix = *rpDetail.AzureBackupRGNamePrefix
	}
	block["prefix"] = prefix

        suffix := ""
	if rpDetail.AzureBackupRGNameSuffix != nil {
		suffix = *rpDetail.AzureBackupRGNameSuffix
	}
	block["suffix"] = suffix
```